client/src/api/results.js

```diff
@@ -1,148 +1,124 @@
+// Update offer with reserved_by user_id
+export async function reserveOffer(offerId) {
+  try {
+    const response = await fetch(`/api/offers/${offerId}`, {
+      method: "PATCH",
+      headers: {
+        "Content-Type": "application/json",
+      },
+    });
+    const result = await response.json();
+    return result;
+  } catch (error) {
+    alert(error.message);
+    return null;
+  }
 }
 
-export async function fetchOffer() {
-  const mockedResult = {
-    id: 1,
-    title: "12 frische Bananen",
-    category: "fruits",
-    tags: ["vegan", "vegetarisch", "laktosefrei", "glutenfrei"],
-    location: {
-      name: "Papa",
-      street: "Musterstraße 12",
-      zip: "50968",
-      city: "Köln",
-    },
-    date: "15.08.2020",
-    time: "15:00 - 16:00",
-  };
-  return mockedResult;
+export async function fetchReservations() {
+  try {
+    const response = await fetch("/api/offers/user/reservations", {
+      method: "GET",
+      headers: {
+        "Content-Type": "application/json",
+      },
+    });
+    const result = await response.json();
+    console.log(result);
+    return result;
+  } catch (error) {
+    alert(error.message);
+    return null;
+  }
 }
 
-export async function fetchReservations(userId) {
-  const mockedResults = [
-    {
-      id: 1,
-      title: "Bananen",
-      category: "fruits",
-      city: "50968 Köln",
-      date: "15.08.2020",
-      time: "15:00 - 16:00",
-    },
-    {
-      id: 2,
-      title: "Joghurt",
-      category: "milk",
-      city: "50968 Köln",
-      date: "15.08.2020",
-      time: "14:00 - 15:00",
-    },
-    {
-      id: 3,
-      title: "Baguette",
-      category: "bread",
-      city: "50968 Köln",
-      date: "15.08.2020",
-      time: "14:00 - 15:00",
-    },
-  ];
-  return mockedResults;
+export async function fetchOffers() {
+  try {
+    const response = await fetch("/api/offers/user/offers", {
+      method: "GET",
+      headers: {
+        "Content-Type": "application/json",
+      },
+    });
+    const result = await response.json();
+    console.log(result);
+    return result;
+  } catch (error) {
+    alert(error.message);
+    return null;
+  }
 }
-export async function fetchOffers(userId) {
-  const mockedResults = [
-    {
-      id: 2,
-      title: "Joghurt",
-      category: "milk",
-      city: "50968 Köln",
-      date: "15.08.2020",
-      time: "14:00 - 15:00",
-    },
-    {
-      id: 3,
-      title: "Baguette",
-      category: "bread",
-      city: "50968 Köln",
-      date: "15.08.2020",
-      time: "14:00 - 15:00",
-    },
-  ];
-  return mockedResults;
+
+// Create Offer
+export async function createOffer(
+  data,
+  formattedDate,
+  start_time,
+  end_time,
+  tags
+) {
+  try {
+    const request = {
+      title: data.title,
+      category: data.category,
+      location: {
+        name: data.name,
+        street: data.street,
+        city: data.city,
+        zip: data.zip,
+      },
+      start_time: start_time,
```

Comment:
you use camelCase in your other code, why start with underscores?

Reply:
Will change it in further steps. Thank you!